### DISCUSSION TITLE
RTX video enhancement is enabled by default.

### DISCUSSION THREAD

**Discussion by rgerg34g3g** [mifi/lossless-cut#1909] [Ideas]
RTX video enhancement is enabled by default. Would be nice to have an on/off toggle in the settings, because I don't need the upscaling (it increases power consumption) when I'm just cutting videos, and having to go to NVIDIA control panel to enable/disable it globally is a bit annoying.
![S5Q3fTMgrm](https://github.com/mifi/lossless-cut/assets/98705916/0cf4039a-715b-4509-9872-916be6678ab0)

**Comment by mifi** [comment]
I have no idea how to do that. I don't think losslesscut even has control over that, as it seems to be an NVIDIA specific thing.

**Reply by rgerg34g3g** [reply]
Individual programs can control it, there is a setting for it in PotPlayer called "D3D11 GPU Super Resolution".

**Reply by mifi** [reply]
unfortunatley i have no idea how to do that. and potplayer is not open source so it's not possible to see how it's done

**Reply by rgerg34g3g** [reply]
That's pretty funny actually, because when the feature came out, video players were pretty slow to adopting it, and to this day only a few support it, but somehow your program supports it without you even knowing what it is :D

**Reply by mifi** [reply]
oh probably because it's based on electron which is based on chromium which is basically google chrome
